src/ktn_recipe.erl

```diff
@@ -22,6 +22,9 @@
 -type transitions()            :: [step()].
 -type normalized_transitions() :: [transition()].
 
+-export_type([transitions/0]).
```

Comment:
Let me know if you'd prefer to not do this and simply adjust tests to not depend on an external type.

Reply:
This is fine 👌